storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -3272,6 +3272,12 @@ xb_load_single_table_tablespace(
 	const char *filname,
 	bool is_remote)
 {
+	/* Ignore .isl files on XtraBackup recovery. All tablespaces must be
+	local. */
+	if (is_remote && !srv_backup_mode) {
```

Comment:
after adding this condition, should we also remove similar one in `xb_load_single_table_tablespaces`?